content/usage/secrets.md

```diff
@@ -99,7 +101,9 @@ Unable to validate Yaml checksum.
 
 # Pull Requests
 
-Secret variables are not injected into to the `build` section of the `.drone.yml` if your repository is public and the build is a pull request. This is for security purposes to prevent a malicious pull request from leaking your secrets.
+Secret variables are also injected into to the `build` and `compose` sections of the `.drone.yml` for pull requests by default if the `.drone.sec` checksum matches. If you need pull requests to exhibit different behavior for security purposes, utilize the [condition](http://readme.drone.io/usage/build_test/#conditions:fb92aa3346185c57f15afda861d465a3) `pull_request` event. This can prevent a malicious pull request from leaking your secrets.
```

Comment:
I'm not super familiar with Hugo but is there a way to get that link using it? Seems like the link was just copied rather than being something hugo can generate.